README.rst

```diff
@@ -33,11 +36,66 @@ Once Redis is installed:
     make serve
 
 
-Configuration can be changed to persist everything in memory (not
-recommended). To do that, `conf/cliquet.ini` file should have the
-following config::
+Storage backend
+===============
+
+Configuration can be changed to persist the records in different storage engines.
+
+
+In-Memory
+---------
+
+Useful for development or testing purposes, but records are lost after each server restart.
+
+In configuration::
+
+    cliquet.storage_url.storage_backend = cliquet.storage.memory
+
+
+Redis
+-----
+
+Useful for very low server load, but won't scale since records sorting and filtering
+are performed in memory.
+
+In configuration::
+
+    cliquet.storage_backend = cliquet.storage.redis
+
+*(Optional)* Instance location URI can be customized::
 
-    cliquet.storage_backend = cliquet.storage.memory
+    cliquet.storage_url = localhost:6379/0
```

Comment:
I think we should add a scheme instead, something like: ``redis://localhost:6379/0``